### PR TITLE
Update snap binary names to snaptel/snapteld

### PR DESCRIPTION
**Pull request by mkleina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,15 +39,15 @@ This builds the plugin in `/build/rootfs/`
 ### Run
 Run the snap agent with the config file
 
-> $GOPATH/bin/snap-v0.14.0-beta/bin/snapd --plugin-trust 0 --log-level 1 --config $GOPATH/src/github.com/intelsdi-x/snap-plugin-collector-nginx/config.json
```
Comment (IzabellaRaulin) [inline README.md:42]: Is it possible not to refer to version 0.14?

Comment (mkleina) [reply README.md:42]: I replaced $GOPATH with $SNAP_PATH and removed reference to exact version completely, is it right?

**Review by katarzyna-z — APPROVED** [review]
LGTM

**Review by IzabellaRaulin — APPROVED** [review]
LGTM too

**Pull request merged**